### PR TITLE
git-log: add German translation

### PR DESCRIPTION
**Pull request by CraxyTM**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Hi,

I've added a German translation for the git-log page.

### CODE REVIEW

File: pages.de/common/git-log.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# git log
+
+> Zeigt die Commit-Historie an.
```
Comment (navarroaxel) [inline pages.de/common/git-log.md:3]: The verbs should be in `present indicative`, not `present continuous`. I don't speak German but I should check.

**Review by navarroaxel — APPROVED** [review]
Hi @CraxyTM, welcome and thanks for your submission. I left 1 question.

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

I don't know what the `present indicative` is, so I can't comment on that.

File: pages.de/common/git-log.md (hunk 1 above)
Comment (navarroaxel) [reply pages.de/common/git-log.md:3]: @LukBukkit, based on your PR #4793. Should we replace `Zeigt` by `Zeige` here?

Comment (LukBukkit) [reply pages.de/common/git-log.md:3]: I'm sorry for my late answer. 
Yeah, I would prefer the usage of `Zeige` as it is the [imperative of `Zeigen`](https://www.duden.de/rechtschreibung/zeigen#grammatik) and so it does sound more targeted to the user. A lot of other German translations are also using the imperative:
* [`7z`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/7z.md)
* [`bash`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/bash.md)
* [`bat`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/bat.md)

But not every translated pages does, for example the page [`git`](https://github.com/tldr-pages/tldr/blob/master/pages.de/common/git.md) also uses `Zeigt`.

Comment (navarroaxel) [reply pages.de/common/git-log.md:3]: @CraxyTM Could you please apply this change?

Comment (sbrl) [reply pages.de/common/git-log.md:3]: Hrm. If other translations are using the imperative, then it's probably fine to use the imperative here too? I mean I don't speak German and I have no idea what the imperative, present simple, or indicative are, so I'm not sure can comment here.

Comment (CraxyTM) [reply pages.de/common/git-log.md:3]: I replaced "Zeigt" with "Zeige" as requested by @navarroaxel

**Pull request merged**